packages/block-library/src/navigation-link/edit.js

```diff
@@ -216,24 +152,27 @@ function NavigationLinkEdit( {
 						placeholder={ itemLabelPlaceholder }
 						withoutInteractiveFormatting
 					/>
+					{
+						/**
+						 * The  isSelected check shouldn't be necessary but due to a focus loss happening
+						 * when selecting a suggestion in the link popover, we force close on block unselection.
+						 */
+					}
```

Comment:
Is this comment still valid? Looking at the conversation from https://github.com/WordPress/gutenberg/pull/19396#discussion_r362773451, I'm wondering if this might have since been updated? Otherwise, I don't immediately see the `isSelected` check that this is referring to.

Reply:
yes, it's necessary but should moved (it's done by the hook now)